### PR TITLE
Ruby group capstone project submission

### PR DESCRIPTION
**Pull request by utsargo**
### Description

- Create a main.rb file that will serve as your console app entry-point.
- In console app, at the start:
    - Presents the user with a list of options to perform.
    - Let users choose an option.
    - If needed, ask for parameters for the option.
    - Have a way to quit the app.
- The following options are available:
    - List all books
    - List all music albums
    - List all movies
    - List of games
    - List all genres (e.g 'Comedy', 'Thriller')
    - List all labels (e.g. 'Gift', 'New')
    - List all authors (e.g. 'Stephen King')
    - List all sources (e.g. 'From a friend', 'Online shop')
    - Add a book
    - Add a music album
    - Add a movie
    - Add a game
  - All data should be preserved by saving collections in .json files.
  
### Team
- @utsargo 
- @tsohleDev 
- @asifaminisonline 

### Video Documentation
https://drive.google.com/file/d/14y3_FvNJBw9sX3BsDBy_NpKcx5ptTDqN/view?usp=share_link

### CODE REVIEW

File: main.rb [hunk 1]
```diff
@@ -0,0 +1,130 @@
...
+  end
+
+  def list_authors
+    Book.list_authors
+  end
+
+  def add_book_item
+    ItemData.add_item('book', Book.create_from_ui)
+    puts 'Book added'
+  end
```
Comment (Mihndim2020) [inline main.rb:54]: - [ ] I am unable to add a book, please checkout the screenshot below and kindly fix the issue. 
add_book_error

File: main.rb [hunk 2]
```diff
@@ -0,0 +1,130 @@
...
+
+  def add_music_album_item
+    ItemData.add_item('music_album', MusicAlbum.create_from_ui)
+    puts 'Music album added'
+  end
```
Comment (Mihndim2020) [inline main.rb:59]: - [ ] I am unable to add a music album, please checkout the screenshot below and kindly fix the issue. 

add_music_error

File: main.rb [hunk 3]
```diff
@@ -0,0 +1,130 @@
...
+
+  def add_game_item
+    ItemData.add_item('game', Game.create_from_ui)
+    puts 'Game added'
+  end
```
Comment (Mihndim2020) [inline main.rb:64]: - [ ] I am unable to add a game, please checkout the screenshot below and kindly fix the issue. 

add_game_error

**Review by Mihndim2020 — CHANGES_REQUESTED** [review]
### Hi Team, 

### Good job so far!
### To Highlight! :clap:  :green_circle: 

:heavy_check_mark:  Good commit messages
:heavy_check_mark: Descriptive Readme file
:heavy_check_mark: Clean and Organized code
:heavy_check_mark: Descriptive `PR` summary

There are some issues that you still need to work on to get this project approved, but you are almost there!
ALMOST-MINION

# Status: Requires Changes ♻️
_Check the comments below and under the review._

- [x] As it is a [requirement ](https://github.com/microverseinc/curriculum-ruby/blob/main/group-capstone/catalog_of_my_things.md)for this project, 
 
#### Project documentation

- You should also highlight some interesting piece of code or something you built that you are very proud of.
- You all should appear in the video and talk about the project.

##### Please make sure each member of the project appears in the presentation video and talk about their own contribution in the project, what they have done, something they have built that they are proud of, and their cameras should be `ON` during the presentation. Please try to distribute the five minutes evenly among the three members. 


## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

### N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear or you can get to me on slack  using this handle @mihndim.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: main.rb (hunk 1 above)
Comment (utsargo) [reply main.rb:54]: @Mihndim2020 Thanks for pointing this out. There was some old code in the main.rb.

**Review by codecaiine — APPROVED** [review]
Hi @utsargo @tsohleDev @asifaminisonline , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
- Clean code ✔️ 
- App works as expected ✔️ 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me (@codecaiine) in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**